exalearn-ui/src/app/components/timer/timer.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+
+// Test time is set to 1 hour (in seconds)
+const TEST_DURATION = 3600;
```

Comment:
We need to make component flexible and give the ability to set the time duration through a custom attribute
(Just put at least TODO: ... to make it later)